styleguide/derek/examples/subpanels.ejs

```diff
@@ -0,0 +1,158 @@
+<%
+
+// List of available layouts.
+var layouts = {
+  '50_50': {
+    'title': '50/50 Layout',
+    'class_left': 'col-md-6',
+    'class_right': 'col-md-6'
+  },
+  '67_33': {
+    'title': '67/33 Layout',
+    'class_left': 'col-md-8',
+    'class_right': 'col-md-4'
+  }
+};
+
+// List of available patterns sorted by title.
+var patterns = {
+  'announcements': {'title': 'Announcements'},
+  'additional-resources': {'title': 'Additional Resources'},
+  'business-resources': {'title': 'Business Resources'},
+  'cta': {'title': 'CTA'},
+  'features': {'title': 'Features'},
+  'services-3-col': {'title': 'IAS: 3 cols'},
+  'services-4-col': {'title': 'IAS: 4 cols'},
+  'services-5-col': {'title': 'IAS: 5 cols'},
+  'services-6-col': {'title': 'IAS: 6 cols'},
+  'key-offers': {'title': 'Key Offers'},
+  'lead': {'title': 'Lead'},
+  'list': {'title': 'List'},
+  'testimonials': {'title': 'Quote'},
+  'text': {'title': 'Text'},
+  'value-prop': {'title': 'Value Prop'}
+};
+%>
+
+<div class="container-fluid">
+  <div class="row standard-padding-full center-block">
+    <div class="col-md-12">
+      <h5 class="red">Global Ecomm Subpanels</h5>
+      <h2 class="no-margin">Yo dawg, I heard you like panels, so we put panels inside of panels.</h2>
+    </div>
+  </div>
+</div>
+
+<div id="pattern-selector" class="bg-steel-blue container-fluid half-padding-full">
+  <div class="row center-block">
+    <div class="col-sm-2">
+      <select id="mp-layout" name="layout">
+        <% for (var layout in layouts) { %>
+        <option value="<%- layout %>"><%- layouts[layout].title %></option>
+        <% } %>
+      </select>
+    </div>
+    <div class="col-sm-3">
+      <select id="mp-pattern-left" name="pattern_left">
+        <% for (var pattern in patterns) { %>
+        <option value="<%- pattern %>">[Left] <%- patterns[pattern].title %></option>
+        <% } %>
+      </select>
+    </div>
+    <div class="col-sm-3">
+      <select id="mp-pattern-right" name="pattern_right">
+        <% for (var pattern in patterns) { %>
+        <option value="<%- pattern %>">[Right] <%- patterns[pattern].title %></option>
+        <% } %>
+      </select>
+    </div>
+    <div class="col-sm-4">
+      <input type="submit" class="btn btn-xs btn-default" id="search-mini-panels" value="Go to Panel"/>
+      <input type="reset" class="btn btn-xs btn-info" id="reset-mini-panels" value="Back to Top"/>
+    </div>
+  </div>
+</div>
```

Comment:
This pattern selector provides a way for users to select a specific layout/left pattern/right pattern.  There is corresponding javascript for this at the bottom of the file.